core/indigo-core/common/math/algebra.h

```diff
@@ -303,6 +303,54 @@ namespace indigo
             _rightTop = other._rightTop;
         }
 
+        inline bool pt_in_rect(const Vec2f& pt) const
+        {
+            return _leftBottom.x < pt.x && _leftBottom.y < pt.y && _rightTop.x > pt.x && _rightTop.y > pt.y;
+        }
+
+        inline bool ray_intersects_rect(const Vec2f& begin, const Vec2f& end)
```

Comment:
Looking at other names, here was used camelCase style, not Snake case. Let's follow the style of the header.

Reply:
fixed